modules/installation-creating-azure-control-plane.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-creating-azure-control-plane_{context}"]
+= Creating the control plane machines in Azure
+
+You must create the control plane machines in Microsoft Azure for your cluster
+to use. One way to create these machines is to modify the provided Azure
+Resource Manager (ARM) template.
+
+[NOTE]
+====
+If you do not use the provided ARM template to create your control plane
+machines, you must review the provided information and manually create the
+infrastructure. If your cluster does not initialize correctly, you might have to
+contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure an Azure account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VNet and assoicated subnets in Azure.
+* Create and configure networking and load balancers in Azure.
+* Create control plane and compute roles.
+* Create the bootstrap machine.
+
+.Procedure
+
+. Copy the template from the *ARM template for control plane machines*
+section of this topic and save it as `05_masters.json` on your computer.
+This template describes the control plane machines that your cluster requires.
+
+. Export the following variable needed by the control plane machine deployment:
++
+----
+$ export MASTER_IGNITION=`cat master.ign | base64`
+----
+
+. Create the deployment by using the `az` CLI:
++
+----
+$ az group deployment create -g ${RESOURCE_GROUP} \
```

Comment:
It looks like we are missing the last lane here

`az group deployment create -g $RESOURCE_GROUP \
  --template-file "05_masters.json" \
  --parameters masterIgnition="$MASTER_IGNITION" \
  --parameters sshKeyData="$SSH_KEY" \
  --parameters privateDNSZoneName="${CLUSTER_NAME}.${BASE_DOMAIN}" \
  --parameters baseName="$INFRA_ID"`

Reply:
Good catch; I'll add that last line with the `$INFRA_ID`.